src/tests/test_journal_vdev.cpp

```diff
@@ -315,20 +360,105 @@ class VDevIOTest : public ::testing::Test {
     }
 
 private:
+    logdev_id_t m_logdev_id = 0;
     off_t m_start_off = 0;
     uint64_t m_wrt_cnt = 0;
     uint64_t m_read_cnt = 0;
     uint64_t m_truncate_cnt = 0;
     uint64_t m_truncate_loop_back_cnt = 0;
-    uint64_t m_total_size = 0;
     std::map< off_t, write_info > m_off_to_info_map;
     Clock::time_point m_start_time;
-    std::shared_ptr< JournalVirtualDev > m_vdev;
+    std::shared_ptr< JournalVirtualDev::Descriptor > m_vdev_jd;
+    friend class VDevJournalIOTest;
 };
 
-TEST_F(VDevIOTest, VDevIOTest) { this->execute(); }
+// TODO add more tests covering unclean shutdown and more corner cases.
+
+TEST_F(VDevJournalIOTest, LongRunning) {
```

Comment:
Can we also specify how long has it been run in the PR description, would be helpful for reviewers?

It should be good to also add this long running test case to existing logstore long running, right?

Reply:
Currently its running just 40secs. Yes we should add it to long running.